### ISSUE TITLE
App crashes when anime video is loading

### ISSUE THREAD

**Issue by H3LLB0Y101** [jmir1/aniyomi#212]
### Steps to reproduce

opens aniyomi
click anime
clicks episode
episode is loading
"Aniyomi had stopped unexpectedly, Close app"
presses close app
it takes me back to the list of episodes

### Expected behavior

Video should be played and app should not crash

### Actual behavior

The app crashes

### Aniyomi version

0.12.3.1

### Android version

9.0

### Device

Asus ROG 2

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/jmir1/aniyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.3.1](https://github.com/jmir1/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by jmir1** [comment]
go to settings - advanced - dump crash logs and please send those, otherwise, i won't know what the issue is :)

**Comment by kyoyacchi** [comment]
I have the same problem as well.


Here's my crash log:


```--------- beginning of crash
11-07 22:44:06.298  6115  6115 E AndroidRuntime: FATAL EXCEPTION: main
11-07 22:44:06.298  6115  6115 E AndroidRuntime: Process: xyz.jmir.tachiyomi.mi.debug, PID: 6115
11-07 22:44:06.298  6115  6115 E AndroidRuntime: java.lang.NullPointerException
11-07 22:44:06.298  6115  6115 E AndroidRuntime: 	at eu.kanade.tachiyomi.ui.player.PlayerActivity$onGetLinks$1.invokeSuspend(PlayerActivity.kt:12)
11-07 22:44:06.298  6115  6115 E AndroidRuntime: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:3)
11-07 22:44:06.298  6115  6115 E AndroidRuntime: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:18)
11-07 22:44:06.298  6115  6115 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:790)
11-07 22:44:06.298  6115  6115 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
11-07 22:44:06.298  6115  6115 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
11-07 22:44:06.298  6115  6115 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6687)
11-07 22:44:06.298  6115  6115 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11-07 22:44:06.298  6115  6115 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
11-07 22:44:06.298  6115  6115 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:810)
--------- beginning of system
App version: 0.12.3.2-4285 (standard, d7121599f, 71, 2021-10-30T22:33Z)
Android version: 8.1.0 (SDK 27)
Android build ID: OPM1.171019.026
Device brand: lge
Device manufacturer: LGE
Device name: mh
Device model: LG-M700
Device product name: mh_global_com```

**Comment by kyoyacchi** [comment]
I have the same problem as well.

Here's my crash log:
@jmir1 

 
[tachiyomi_crash_logs.txt](https://github.com/jmir1/aniyomi/files/7492986/tachiyomi_crash_logs.txt)

**Comment by H3LLB0Y101** [comment]
> go to "settings - advanced - dump crash logs" and please send those, otherwise, i won't know what the issue is :)
[tachiyomi_crash_logs.txt](https://github.com/jmir1/aniyomi/files/7572287/tachiyomi_crash_logs.txt)

**Comment by jmir1** [comment]
Fixed by 1a0f90793a72b9930c5d9842f978b8b4354a2982
